course/test/src/isometry_TEST.cc

```diff
@@ -40,8 +40,8 @@ GTEST_TEST(Vector3Test, Vector3Operations) {
   EXPECT_EQ(ss.str(), "(x: 1, y: 2, z: 3)");
 
   EXPECT_TRUE(Vector3::kUnitX == Vector3(1., 0., 0));
-  EXPECT_TRUE(Vector3::kUnitX != {1., 1., 0});
-  EXPECT_TRUE(Vector3::kUnitY == {0., 1., 0});
+  EXPECT_TRUE(Vector3::kUnitX != Vector3(1., 1., 0));
```

Comment:
Instead of creating a `Vector3` create an `initializer_list`, which was the original idea. Same below.